### DISCUSSION TITLE
allocated at ???

### DISCUSSION THREAD

**Discussion by EralpB** [bloomberg/memray#70] [Q&A] [answered]
Hello I'm profiling an ETL written with pandas, my peak usage is 18 gb and 17 GB of it is 1 line "allocated at ???" this is not helpful at all, is this expected? how can I profile it better?

reports by "PYTHONMALLOC=malloc" or "--leaks" doesn't change much.

I can't share the result HTML because of safety reasons. but I wonder if this is a common phenomena or not.

**Comment by pablogsal** [comment, marked as answer]
This happens if you are using `--native` and the libraries you are using don't have debug symbols available so it is not possible for us to resolve the name of the function where that happens. This can happen as well if you generate the binary dump in one machine and generate the flamegraph in another, because the second one won't have the shared objects, libraries or debug symbols that are necessary.

Unfortunately there is nothing we can do on our side :(
